src/aesthetics.jl

```diff
@@ -272,10 +273,10 @@ cat_aes_var!(a, b) = a
 
 function cat_aes_var!(a::AbstractArray{T}, b::AbstractArray{U}) where {T, U}
     V = promote_type(T, U)
-    if isa(a, DataArray) || isa(b, DataArray)
-        ab = DataArray(V, length(a) + length(b))
+    if isa(a, Array{Union{Missing,T}}) || isa(b, Array{Union{Missing,U}})
```

Comment:
`Array{V}(undef, length(a) + length(b))` should now cover both branches (`Union{Missing,V} == V` in the first branch).